src/components/Modal.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div
+    id="modal-fadeInScale-fs"
+    :class="{
+      'modal is-active': isActive,
+       'modal': !isActive,
+    }">
+    <div class="modal-background"></div>
+    <div class="modal-content">
+         <button
+      class="modal-button-close delete is-large"
+        aria-label="close"
+        v-on:click="handleModalClose()"
+      ></button>
+      <div class="has-text-centered">
+        <h2>{{ team.project.name }}</h2>
+        <p>{{ team.project.description }}</p>
+
+        <p class="socials">
+          <a :href="team.project.links.repository" target="_blank">
+           <i class="fab fa-github fa-2x"></i>
+          </a>
+          <a :href="team.project.links.website || placeholderDeployment" target="_blank">
+            <i class="fas fa-desktop fa-2x"></i>
+          </a>
+          <a :href="team.project.links.writeup || placeholderMedium" target="_blank">
+            <i class="fab fa-medium-m fa-2x"></i>
+          </a>
+        </p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { Team } from '@/data/types';
+export default Vue.extend({
+  name: 'Modal',
```

Comment:
to align with the name of the card component, maybe something more descriptive like:

```suggestion
  name: 'TeamProjectModal',
```

and similar change for the filename?

Reply:
Another thing that I thought of was to make the Modal more generic so we can use the same component in different scenarios. However that would need some code refactoring. For now I am open to making this a more specific component in the interest of time so I'll change the name.

Reply:
Fixed in: c525c21